basics/task7/index.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+session_start();
```

Comment:
This looks like the login form. Why are we starting session when the user is not even logged in ?

Reply:
To check if the user is already logged in or not. If the user is already logged in and tries to access the login page, then the user will be automatically redirected to the homepage.